bindings/java/src/org/sleuthkit/datamodel/CommunicationsManager.java

```diff
@@ -1409,16 +1379,102 @@ private String normalizePhoneNum(String phoneNum) {
 	 * Utility method to convert a list to an CSV string
 	 */
 	private static <T> String buildCSVString(Collection<T> values) {
-		if (values==null || values.isEmpty()) return "";
+		if (values==null || values.isEmpty()) 
+			return "";
+		
 		StringBuilder result = new StringBuilder();
 		for (T val : values) {
 			result.append(val);
 			result.append(",");
 		}
+		
 		return result.substring(0, result.length() - 1);
 	}
 	
-	/*
+	/**
```

Comment:
It looks like all of these SQL helper methods can be static.

Reply:
Further downstream, both the getSQL() methods need to access non static methods (to convert device_ids to ds_object_ids, and to convert Account>type to account_type_id, respectively) 

That's why these cant be static.

Reply:
Ok